src/platform/forms/tests/save-in-progress/RoutedSavableReviewPage.unit.spec.jsx

```diff
@@ -227,7 +227,7 @@ describe('Schemaform save in progress: RoutedSavableReviewPage', () => {
 
     let tree;
     beforeEach(() => {
-      tree = shallow(
+      tree = mount(
```

Comment:
how can I go about testing this feature without a full mount? Not sure how to check to see if children are present without a full mount...